src/xia2/cli/cluster_analysis.py

```diff
@@ -268,6 +268,11 @@ def run(args=sys.argv[1:]):
             f.write(html.encode("utf-8", "xmlcharrefreplace"))
 
         logger.info(f"Clusters recommended for comparison in {params.output.log}")
+        logger.info("----------------")
+        logger.info("Output given as DIALS .expt/.refl files:")
+        logger.info("To merge rotation data: use dials.merge")
+        logger.info("To merge still data: use xia2.ssx_reduce")
```

Comment:
```suggestion
        logger.info("To merge still data: use xia2.ssx_reduce with the option steps=merge")
```
I think you are suggesting that the data from each cluster should be merged without further processing? If so it would be helpful to add this bit of text to avoid accidental rescaling of individual clusters.